### PR TITLE
PORT-10620 Restructure Pagerduty Integration Docs

### PR DESCRIPTION
**Pull request by kodjomiles**
# Description

Restructure the Pagerduty integration guide to follow new integration docs structure. 

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- PagerDuty Integration Docs Page(`/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty`)

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty.md [hunk 1]
```diff
@@ -20,16 +20,26 @@ import OceanSaasInstallation from "/docs/build-your-software-catalog/sync-data-t
 
 Port's PagerDuty integration allows you to import `schedules`, `oncalls`, `services`, `incidents` and `escalation_policies` from your PagerDuty account into Port, according to your mapping and definitions.
 
-## Common use cases
+<Prerequisites />
```
Comment (PeyGis) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty.md:23]: I think the prerequisites should come under the installation section

**Review by PeyGis — CHANGES_REQUESTED** [review]
looks great. One of the section that needs to be removed is the let's test it section. see below.
from the new integration template
```
# Examples

<!-- Make sure to add examples of supported blueprints and mappings -->
<!--If there are 5 or more examples, create a new page for the examples and link to it here. -->

To view and test the integration's mapping against examples of the third-party API responses, use the jq playground in your [data sources page](https://app.getport.io/settings/data-sources). Find the integration in the list of data sources and click on it to open the playground.
```


Screenshot 2024-09-27 at 4 13 24 PM

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty.md (hunk 1 above)
Comment (kodjomiles) [reply docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty.md:23]: done

**Review by PeyGis — APPROVED** [review]
LGTM

File: docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty/pagerduty.md [hunk 2]
```diff
@@ -390,89 +406,14 @@ ingest_data:
...
-          mappings: ...
-  ```
+The mapping makes use of the [JQ JSON processor](https://stedolan.github.io/jq/manual/) to select, modify, concatenate, transform and perform other operations on existing fields and values from the integration API.
 
-  :::tip Blueprint key
-  Note the value of the `blueprint` key - if you want to use a hardcoded string, you need to encapsulate it in 2 sets of quotes, for example use a pair of single-quotes (`'`) and then another pair of double-quotes (`"`)
-  :::
 
-### Ingest data into Port
+:::info Ingest data into Port
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/incident-management/pagerduty/pagerduty.md:416]: we can remove this admonition, like in other integrations

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**